catalog-info.yaml

```
@@ -0,0 +1,28 @@
+---
+ # yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+ apiVersion: backstage.io/v1alpha1
+ kind: Resource
+ metadata:
+   name: buildkite-pipeline-package-registry
+   description: Buildkite Pipeline for package-registry
+   links:
+     - title: Pipeline
+       url: https://buildkite.com/elastic/package-registry
+
+ spec:
+   type: buildkite-pipeline
+   owner: group:ingest-fp
+   system: buildkite
+   implementation:
+     apiVersion: buildkite.elastic.dev/v1
+     kind: Pipeline
+     metadata:
+       name: package-registry
+     spec:
+       repository: elastic/package-registry
+       pipeline_file: ".buildkite/pipeline.yml"
+       teams:
+         ingest-fp:
+           access_level: MANAGE_BUILD_AND_READ
+         everyone:
+           access_level: READ_ONLY
```

As this repository is managed mainly by ecosystem team I would also include this:

```suggestion
       teams:
         ecosystem: {}
         ingest-fp:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
```

An empty map is like MANAGE_BUILD_AND_READ, so it could be set like that:

> MANAGE_BUILD_AND_READ (empty brackets symbol): Allows a team to view and create builds. Also, edit pipeline settings.

https://github.com/elastic/ci/blob/main/docs/buildkite/getting_started.mdx

Maybe ingest-fp should just have BUILD_AND_READ @alexsapran So not everyone can edit the pipeline settings

> BUILD_AND_READ: can view and create builds but cannot edit pipeline settings.

---

Let's not modify ingest-fp team permissions for now. Let's keep MANAGE_BUILD_AND_READ